pynestml/grammars/PyNestMLParser.g4

```diff
@@ -233,9 +233,7 @@ parser grammar PyNestMLParser;
     @attribute function: A block declaring a user-defined function.
   */
   neuronBody: COLON
-         (NEWLINE | blockWithVariables | equationsBlock | inputBlock | outputBlock | updateBlock | function)*
-         END_KEYWORD;
-
+         NEWLINE INDENT ( blockWithVariables | equationsBlock | inputBlock | outputBlock | updateBlock | function)+ DEDENT;
```

Comment:
Could we not have an empty neuron? Maybe for testing (don't know if this is very useful...) Same comment goes for blocks etc. in the grammar rules below.

Reply:
I guess not, as it is difficult to know the indentation level without any statements in the blocks. Python also does not allow empty blocks. Should we support it?

Reply:
Couldn't we just add the [`pass`](https://docs.python.org/3/tutorial/controlflow.html#pass-statements) statement from Python to indicate that a block is not doing anything?

Reply:
Thanks, I'm fine with this as-is.